ui/web/static/js/charts.js

```diff
@@ -106,13 +70,26 @@ function createAlarmsTable(chartData, config) {
             </tbody>
           </table>
         </div>
+    </div>
       `;
-  return config;
+  }
 }
 
-function createAreaLineChart(chartData, config) {
-  config.Script = `
-        var areaLineChart = echarts.init(document.getElementById("${config.ID}"));
+class AreaLineChart {
+  constructor(chartData, widgetID) {
+    this.chartData = chartData;
+    this.ID = widgetID;
+    this.Style = {
+      width: "450px",
+      height: "450px",
+    };
+    let styleString = `width: ${this.Style.width}; height: ${this.Style.height};`;
+    this.Content = `
+    <div class="item-content" id="${this.ID}" style="${styleString}">
```

Comment:
```suggestion
    <div class="item-content" id="${this.ID}" style="[${styleString}](width: ${this.Style.width}; height: ${this.Style.height};)">
```
No need to declare this as a separate variable